pkg/stores/proxy/unformatter_test.go

```diff
@@ -0,0 +1,110 @@
+package proxy
+
+import (
+	"testing"
+
+	"github.com/rancher/apiserver/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_unformat(t *testing.T) {
+	tests := []struct {
```

Comment:
It's always good to start a test with `t.Parallel()` and also run subtests in parallel, unless there is global state.

```
for _, test := range tests {
	test := test
	t.Run(test.name, func(t *testing.T) {
		t.Parallel()
```

Reply:
done

Reply:
Nit: the test itself can also be run in parallel:
```suggestion
	func TestUnformat(t *testing.T) {
	    t.Parallel()
```